domains/certificates/RegistryConnector/Worker/Program.cs

```diff
@@ -69,6 +69,7 @@
         });
 
         cfg.ConfigureEndpoints(context);
+        cfg.SetQuorumQueue();
```

Comment:
if(cfg is IRabbitMqEndpointConfigurator rmq)
        rmq.SetQuorumQueue();
        
Would recommend maybe that we make a check if it's rabbitmq. Otherwise it would break if we change to other queue in future.